maas/plugins/README.md

```diff
@@ -335,18 +335,27 @@ connects to an individual member of a galera cluster and checks various statuses
 
 ##### Example Output:
 
-    metric wsrep_replicated_bytes int64 320737952 bytes
-    metric wsrep_received_bytes int64 33470 bytes
-    metric wsrep_commit_window double 1.000000
+    metric wsrep_replicated_bytes int64 0 bytes
+    metric wsrep_received_bytes int64 226600043 bytes
+    metric wsrep_commit_window_size double 1.000161 sequence_delta
     metric wsrep_cluster_size int64 3 nodes
-    metric queries_per_second int64 5792715 qps
-    metric wsrep_cluster_state_uuid string 67e41d08-165d-11e4-9d87-7e94ef43b302
-    metric wsrep_cluster_status string primary
-    metric wsrep_local_state_uuid string 67e41d08-165d-11e4-9d87-7e94ef43b302
-    metric wsrep_local_state_comment string synced
+    metric queries_per_second int64 297004 qps
```

Comment:
it's worth noting that the number that the plugin pulls from mysql is actually a cumulative total of queries since last daemon restart, so this metric is misnamed. We should fix the metric name, and then in the future if we ever need to monitor/alarm on qps we can use the 'rate' function in the alarm language to decipher actual qps.